guarddog/analyzer/sourcecode/exfiltrate-sensitive-data.yml

```diff
@@ -31,6 +31,19 @@ rules:
           - metavariable-regex:
               metavariable: $ENVVAR
               regex: ([\"\'](AWS_ACCESS_KEY_ID|AWS_SECRET_ACCESS_KEY|AWS_SESSION_TOKEN)[\"\'])
+      - patterns:
+          - pattern-inside: |
+              $CONNECT = sqlite3.connect(...)
+              ...
+              $CURSOR = $CONNECT.cursor(...)
+              ...
+              $CURSOR.close(...)
```

Comment:
we don't care about the close, our goal is to require the minimum indicators that conform a detection to avoid overfitting. e.g: in this case the attacker can omit the `.close` statement and you would miss the match